src/xdg-autostart-generator/xdg-autostart-service.c

```diff
@@ -443,7 +443,11 @@ int xdg_autostart_format_exec_start(
                         return log_oom();
 
                 /*
-                 * Expand ~ if it comes at the beginning of an argument to form a path
+                 * Expand ~ if it comes at the beginning of an argument to form a path.
+                 *
+                 * This is not explicitly in the desktop specification,
```

Comment:
Maybe, "explicitly supported" or "explicitly documented"?

Reply:
I'd drop "explicitly"... I mean, it's not "implicitly" in the spec either afaics.

Just say "the specification does not mandate this, but we do it anyway for compatibility"